package.json

```diff
@@ -82,7 +82,7 @@
     "react2angular": "4.0.2",
     "readable-stream": "2.3.3",
     "redux": "3.5.2",
-    "rendition": "4.6.0",
+    "rendition": "^4.27.3",
```

Comment:
Would it make sense to keep this locked (w/o the `^`) as it was before and like the rest dependencies?

Reply:
@thundron @petrosagg @jviotti what do you think?

Reply:
I would say it actually makes sense to unlock it as we will probably make some changes in rendition along the etcher transition to react and that way we are up to date with no fuss.

Reply:
@amdomanska Yes but it's safer to selectively update dependencies like we do for the rest, so should the API change we can detect it more easily and with much more control over the npm-shrinkwrap too.

Reply:
@thundron ok, I will lock it then.
To change it properly: do I have to run usual `npm install -save` and then `npm shrinkwrap --dev`?